Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -4421,25 +4423,19 @@ - (void)setIsViewVisible:(BOOL)isViewVisible
     [self updateCellsVisible];
 }
 
-- (void)setIsAppInBackground:(BOOL)isAppInBackground
-{
-    _isAppInBackground = isAppInBackground;
-
-    [self updateShouldObserveDBModifications];
```

Comment:
Will this present more redrawing to the user since now we're doing it after we become active?

Reply:
I don't think so, but that's definitely something we should verify.